sdks/js/packages/core/react/components/organization/plans/index.tsx

```diff
@@ -283,7 +286,9 @@ const PlanPricingColumn = ({
               />
             ) : value ? (
               <Text>{value}</Text>
```

Comment:
Won't this mean that we show `false` here? Since values are now `false` not `-` in the metadata?

Reply:
if we explicitly define `false` then `false` will come. for empty string or no value means key is not defined in metadata, then `-` will be shown.